src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -871,7 +871,7 @@ def check_connectivity(url='https://azure.microsoft.com', max_retries=5, timeout
 
 def send_raw_request(cli_ctx, method, url, headers=None, uri_parameters=None,  # pylint: disable=too-many-locals,too-many-branches,too-many-statements
                      body=None, skip_authorization_header=False, resource=None, output_file=None,
-                     generated_client_request_id_name='x-ms-client-request-id'):
+                     generated_client_request_id_name='x-ms-client-request-id', log_request_response=True):
```

Comment:
Do we need this `log_request_response` change checked in? If yes, we should keep the default value as `False`

Reply:
By default it will print the entire request and response. Changing the parameter to `disable_request_response_logging` and setting default to `False`